APIs/schemas/receiver_transport_params_mqtt.json

```diff
@@ -0,0 +1,43 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Describes MQTT Receiver transport parameters. The constraints in this schema are minimum constraints, but may be further constrained at the constraints endpoint. Receivers must support at least the `source_ip`, `interface_ip`, `rtp_enabled` and `destination_port` parameters, and must support the `multicast_ip` parameter if they are capable of multicast operation.",
+  "title": "MQTT Receiver Transport Parameters",
+  "type": "array",
+  "items": {
+    "type": "object",
+    "title": "Receiver Input",
+    "properties": {
+      "source_host": {
+        "type": "string",
+        "description": "Hostname or IP hosting the MQTT broker. The Receiver should provide an enum in the constraints endpoint, which should contain 'auto', and the available interface addresses formatted as connection URIs. If the parameter is set to auto the Receiver should establish for itself which broker it should use, based on a discovery mechanism or its own internal configuration.",
```

Comment:
NB, worth remembering that "auto" in IS-05 has to be resolved to the actual value used in the connection at activation time - "auto" should never appear in /active